### PR TITLE
Fix shellcheck warnings

### PR DESCRIPTION
**Pull request by LanderU**

### CODE REVIEW

**Comment by shuhaowu** [comment]
Do you want to run these shellcheck on CI (before the image is built as well)?

File: builder/core.sh [hunk 1]
```diff
@@ -1,11 +1,13 @@
+#!/bin/bash
```
Comment (shuhaowu) [inline builder/core.sh:1]: Technically these are not executable shell scripts as they're meant to be sourced, but I suppose it is OK

**Comment by LanderU** [comment]
@shuhaowu, yeah, I'll include a step in CI to run shelcheck

File: builder/core.sh (hunk 1 above)
Comment (LanderU) [reply builder/core.sh:1]: Yep, I know, but I think it doesn't hurt to add the shebang, if you disagree I can remove it.

Comment (shuhaowu) [reply builder/core.sh:1]: No worries. This is totally fine. :+1:

**Review by shuhaowu — APPROVED** [review]
> yeah, I'll include a step in CI to run shellcheck

Will you do in this PR or another one? Either way, the changes looks good to me. Thanks! :tada:

**Comment by LanderU** [comment]
@shuhaowu, I will open a new one, thanks a lot!

**Pull request merged**